src/ddmd/parse.d

```diff
@@ -21,6 +21,7 @@ import ddmd.root.filename;
 import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 import ddmd.root.rootobject;
+import ddmd.target;
```

Comment:
Not as far as I can see. Listing the dependencies of `ddmd.parse` shows a lot of additional modules compared to before this change:

```
$ dmd -o- -deps=deps.txt -Isrc -Isrc/ddmd -Jres src/ddmd/parse.d && cat deps.txt | grep -v core | grep -v root | cut -f 1 -d ' ' | sort | uniq  | head
ddmd.access
ddmd.aggregate
ddmd.aliasthis
ddmd.apply
ddmd.argtypes
ddmd.arrayop
ddmd.arraytypes
ddmd.astcodegen
ddmd.attrib
ddmd.backend.bcomplex
$ git co 23e15aa1f371343e4dbfad9cb555c90cefbf415a
$ dmd -o- -deps=deps.txt -Isrc -Isrc/ddmd -Jres src/ddmd/parse.d && cat deps.txt | grep -v core | grep -v root | cut -f 1 -d ' ' | sort | uniq
ddmd.entity
ddmd.errors
ddmd.globals
ddmd.id
ddmd.identifier
ddmd.lexer
ddmd.parse
ddmd.tokens
ddmd.utf
```

Note, the first command is with `head`, the second is without `head`.